.gitignore

```diff
@@ -0,0 +1,2 @@
+venv/
```

Comment:
Have you set up the corresponding venv environment with the corresponding dependency?
It would be great to add it in an other commit if you have it.